apps/server/src/modules/deletion/uc/deletion-request.uc.ts

```diff
@@ -104,6 +108,7 @@ export class DeletionRequestUc {
 			]);
 			await this.deletionRequestService.markDeletionRequestAsExecuted(deletionRequest.id);
 		} catch (error) {
+			this.logger.error(`execution of deletionRequest ${deletionRequest.id} was failed`, error);
```

Comment:
Implement the legacy logger to get the .error methods is not the goal of hidding it. Please use the logger and find a way to handle this error.

Reply:
@WojciechGrancow